data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -35,10 +35,13 @@
  * Such properties may be used to provide additional information to Kusto (for example, for the purpose of correlating client/service interaction),
  * may affect what limits and policies get applied to the request, and much more.
  * For a complete list of available client request properties
- * check out https://docs.microsoft.com/en-us/azure/kusto/api/netfx/request-properties#list-of-clientrequestproperties
+ * check out https://docs.microsoft.com/azure/kusto/api/netfx/request-properties#list-of-clientrequestproperties
  */
 public class ClientRequestProperties implements Serializable, TraceableAttributes {
     public static final String OPTION_SERVER_TIMEOUT = "servertimeout";
+
+    // Only use if you understand the ramification
+    public static final String OPTION_CLIENT_MAX_REDIRECT_COUNT = "client_max_redirect_count";
```

Comment:
Add docstring? you can copy-paste the one from .net if you want: `If set and positive, indicates the maximum number of HTTP redirects that the client will process. [Long]`